manage.py

```diff
@@ -11,6 +13,4 @@
     IGNORE_MODULES = 'djangosaml2'
     warnings.filterwarnings("ignore", module=IGNORE_MODULES, category=DeprecationWarning)
 
-    from django.core.management import execute_from_command_line
```

Comment:
I have notices Django by default put the import under `if __name__ == "__main__":` with try/except. I don't remember why the try/except was removed but if this is not causing any trouble should be ok

Reply:
I'm not sure why I moved this, I can move it back.